test/integration/elements/element-with-is.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+/// <reference path="../bower_components/polymer-decorators/global.d.ts" />
+
+@Polymer.decorators.customElement()

```

Comment:
If you're wondering why I did this, its because we would otherwise be trying to redefine a `const customElement` from the other element... an IIFE would work but then you lose the class from global scope.